README.md

```diff
@@ -53,6 +53,21 @@ Please search open issues [here](https://github.com/Azure/azure-service-operator
 
 For more information, see [SUPPORT.md](SUPPORT.md).
 
+## Repository Visualizations
```

Comment:
Wondering if it makes more sense for these links to be in the individual project README's rather than the root one? My thinking with this root README was just as a landing page that answered the "What is the ASO project, what are the versions, which should I choose" and everything else was in the v1 or v2 README (so that people who want v1 didn't have to see v2's stuff and vice versa).

Reply:
I've moved the diagrams somewhat, though not necessarily into README files - as those are often about using the operator, not modifying it. The new locations are better than here, but still not great.

We need to add specific documentation for people who want to do development vs people wanting to run the operator.

Reply:
Yeah good point, if the intended audience is developers (and not users) then CONTRIBUTING or the development.md for v1 is I think a great place to put them. Approved!